hazelcast/src/main/java/com/hazelcast/instance/GroupProperty.java

```diff
@@ -114,14 +114,7 @@
      */
     PERFORMANCE_METRICS_LEVEL("hazelcast.performance.metric.level", ProbeLevel.MANDATORY.name()),
 
-    /**
-     * The delay in seconds between monitor of the performance.
-     * <p/>
-     * The default is 30 seconds.
-     */
-    PERFORMANCE_MONITOR_DELAY_SECONDS("hazelcast.performance.monitor.delay.seconds", 30, SECONDS),
```

Comment:
These should not be officially supported properties. They are there for debugging only